package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
```

Comment:
Because of strange behavior of `prepublish`, maybe consider `prepublishOnly` instead?

https://iamakulov.com/notes/all/npm-4-prepublish/